libraries/AP_HAL_ChibiOS/hwdef/AnyleafH7/defaults.parm

```diff
@@ -0,0 +1,25 @@
+# Onboard ELRS
+SERIAL5_PROTOCOL 23 # RCIN
+# RC_OPTIONS 512  # Suppress ELRS errors
+RC5_OPTION 154 # Set up arm switch to `ArmDisarm with AirMode`
+SERIAL5_BAUD 115
```

Comment:
if this is ELRS you may want the RC_OPTIONS bit 13

Reply:
Thanks; uncommented the commented out RC_OPTIONS 512 line; that sets the suppress ELRS warnings bit.

Reply:
not the same thing

Reply:
bit 13 is needed....and bit 9 as you had....but bit 9 is subjective....bit 13 is required for ELRS